### PR TITLE
feat(vpaas): Send billing id to prosody

### PR DESCRIPTION
**Pull request by vp8x8**
For now we'll send the id both through an explicit http request and through prosody, but once the functionality is implemented on backend we'll drop the first method.

### CODE REVIEW

File: react/features/billing-counter/functions.js [hunk 1]
```diff
@@ -88,3 +87,15 @@ export function getBillingId() {
 
     return billingId;
 }
+
+/**
+ * Returns the billing id for vpaas meetings.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {string}
```
Comment (saghul) [inline react/features/billing-counter/functions.js:95]: nit: this can return undefined too.

**Review by saghul — DISMISSED** [review]

**Comment by vp8x8** [comment]
Jenkins, test this please.

**Comment by vp8x8** [comment]
Jenkins, test this please.

**Comment by vp8x8** [comment]
Jenkins, test this please.

**Comment by vp8x8** [comment]
Jenkins, test this please.

**Review by muscat1 — APPROVED** [review]

**Review by quitrk — APPROVED** [review]

**Pull request merged**